client/extensions/wp-job-manager/components/settings/index.jsx

```diff
@@ -44,6 +45,10 @@ class Settings extends Component {
 
 		return (
 			<Main className={ mainClassName }>
+				<ExtensionRedirect
+					minimumVersion="1.28.0"
```

Comment:
Since we're using `ExtensionRedirect` twice in this extension, do you think it would make sense to store `minimumVersion` and perhaps even `pluginId` elsewhere as a constant so we don't have to keep track of where the redirect is used ?

Reply:
I've moved the minimum plugin version needed in https://github.com/Automattic/wp-calypso/pull/18404/commits/2d7c9f5f9e55c5f5f440a5aecf940d957346b3a8. There are more places where `wp-job-manager` is used than here, so I may do that in a separate PR.